examples/react/w3console/src/share.tsx

```diff
@@ -80,9 +112,43 @@ export function SpaceShare (): JSX.Element {
             value={value}
             onChange={onChange}
           />
-          <a className='w3ui-button text-center block w-52 opacity-30' style={{ opacity: downloadUrl !== '' ? '1' : '0.2' }} href={downloadUrl ?? ''} download={downloadName(downloadUrl !== '', value)}>Download UCAN</a>
+          <a className='w3ui-button text-center block w-52' style={{ opacity: downloadUrl !== '' ? '1' : '0.2' }} href={downloadUrl ?? ''} download={downloadName(downloadUrl !== '', value)}>Download UCAN</a>
```

Comment:
similarly, I'd move this `style=` into the `className`
```suggestion
          <a className={`w3ui-button text-center block w-52 ${downloadUrl !== '' ? 'opacity-100' : 'opacity-20'}`} 
             href={downloadUrl ?? ''} download={downloadName(downloadUrl !== '', value)}>
            Download UCAN
          </a>
```

Reply:
I had an issue with postcss not being able to detect dynamic classNames, which meant the class name wasn't included in the compiled css.